src/components/Skeleton/Skeleton.test.jsx

```diff
@@ -0,0 +1,11 @@
+import { render, screen } from "@testing-library/react";
+import Skeleton from "./Skeleton";
+import { describe, it, expect } from "vitest";
```

Comment:
Usually I like to import external stuff first and then the local ones, vitest should come first